docs/yaml/functions/benchmark.yaml

```diff
@@ -104,3 +104,13 @@ kwargs:
       The starting order of tests with identical priorities is
       implementation-defined. The default priority is 0, negative numbers are
       permitted.
+
+  verbose:
```

Comment:
Does adding the info here mean that https://mesonbuild.com/Reference-manual_functions.html#test gets updated as well?

Reply:
Yes, see [here](https://github.com/mesonbuild/meson/blob/master/docs/yaml/functions/test.yaml):

```yaml
posargs_inherit: benchmark
kwargs_inherit: benchmark
```